types/src/auction/constants.rs

```diff
@@ -5,10 +5,10 @@ use crate::account::AccountHash;
 pub const SYSTEM_ACCOUNT: AccountHash = AccountHash::new([0; 32]);
 
 /// Number of eras before an auction actually defines the set of validators.
-pub const AUCTION_DELAY: u64 = 3;
+//pub const AUCTION_DELAY: u64 = 3;
 
 /// Number of eras to keep track of in past.
-pub const SNAPSHOT_SIZE: usize = AUCTION_DELAY as usize + 1;
+pub const SNAPSHOT_SIZE: usize = 4;
```

Comment:
I'm pretty sure this is incorrect. Possibly SNAPSHOT_SIZE should no longer be a const.

Reply:
Yeah, I've kept it as a placeholder, I wanted just `auction delay` reviewed to see if I had it right, and then roll the changes forward

Reply:
Changed to work chainspec